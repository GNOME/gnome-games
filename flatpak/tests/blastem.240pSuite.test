[Retro Reftest]
Path=/blastem/240pSuite
Core=/app/lib/libretro/blastem_libretro.so
Medias=240pTestSuite-v15.gen;

[Frame 150]
Tests=Run;Video;
Video=blastem.240pSuite.150.png
