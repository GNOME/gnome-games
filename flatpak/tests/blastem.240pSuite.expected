/blastem/240pSuite/Boot: OK
/blastem/240pSuite/Options: OK
/blastem/240pSuite/150/FastForward: OK
/blastem/240pSuite/150/Run: OK
/blastem/240pSuite/150/Video: OK
